src/mode/modeVisual.ts

```diff
@@ -40,6 +40,9 @@ export class VisualMode extends Mode {
     }
 
     shouldBeActivated(key: string, currentMode: ModeName): boolean {
+        if (currentMode !== ModeName.Normal) {
+            return false;
+        }
         return key === "v";
```

Comment:
I'm in the same boat -- learning a lot of little things here and there building out this extension.